server/app/initializers/authority.rb

```diff
@@ -0,0 +1,12 @@
+require 'authority'
+
+Authority.configure do |config|
+  config.abilities =  {
```

Comment:
Is it possible to define abilities per model in Authority? Somehow it feels funny to have global abilities like `assignable` or `deployable`.